src/DynamoCoreWpf/UI/Themes/Modern/DynamoColorsAndBrushes.xaml

```diff
@@ -68,7 +68,11 @@
     <SolidColorBrush x:Key="WorkspaceTabBorderSelectedFalse" Color="#3C3C3C" />
 
     <!--  Nodes  -->
-    <SolidColorBrush x:Key="NodeFrozenOverlayBackground" Color="#BCD3EE" />
+    <SolidColorBrush x:Key="NodeFrozenOverlayColor" Color="#BCD3EE" />
+    <SolidColorBrush x:Key="NodeInfoColor" Color="{StaticResource Blue300}" />
+    <SolidColorBrush x:Key="NodeWarningColor" Color="#FAA21B" />
```

Comment:
when I search the repo for these color strings I find a bunch of them, can you update those usages to point to these new keys?

Reply:
Done!